src/System.Management/tests/System/Management/ManagementObjectTests.cs

```diff
@@ -87,9 +82,6 @@ public void Invoke_Instance_And_Static_Method_Win32_Process()
                 Assert.Equal(0u, resultCode);
 
                 Assert.True(targetProcess.HasExited);
-
-                ManagementException managementException = Assert.Throws<ManagementException>(() => process.Get());
-                Assert.Equal(ManagementStatus.NotFound, managementException.ErrorCode);
```

Comment:
What makes it flaky in this specific test?

Reply:
The suspicion is that the process Id gets re-used so the call `process.Get()` ends up succeeding. That said I don't have confirmation of that, the test never failed on local runs but had a relatively high rate of failure on CI. I am going to give the repro tool a try to see if I can get a repro (out of curiosity to understand the exact cause).